### PR TITLE
Nora Nickels HW 1

### PR DESCRIPTION
**Pull request by nnickels**
Report of Process: To make this file, I first forked the repo for homework 1 from the class page on Git. Then, I opened an RStudio project using the link from this forked repo (cloning the forked repo from my homework repo and opening it in RStudio/Git Client in R, by opening a new project in R). Once the README.md document was opened in RStudio, I made changes in the Markdown window, using the online GitHub guide for RMarkdown. I was able to figure out how to add an image when the image is hyperlinked somewhere online, but couldn't figure out how to do it with a file saved on my computer (even when the file was saved to the same R folder). I saved changes frequently and committed many of these saved changes over the last couple days in the Git Client window in RStudio (commenting on the changes I've made). I then pulled my changes in the Git Client window (to get used to this, although I don't expect outside changes to be made to my homework at this point). Then I push my changes from RStudio, and this sends my commits to Git online. I finally submit my final html and .md files online by creating a pull request to the class homework from which the repo was forked. One issue I came across a few times (both in the terminal and when pushing/pulling from RStudio) was issues "merging" files. In lab on Monday, Josh helped my to resolve this issue basically by telling the computer in terminal where the latest changes were made, but I still worry that I don't fundamentally understand what is going on when I run into this issue. Error messages include statements such as "Pull is not possible because you have unmerged files" and "fatal: refusing to merge unrelated histories".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+https://imb.uchicago.edu/directory/nora-nickels
+
+http://chicago.academia.edu/NoraNickels
+
+## 
+
+> "But with my ego, I could stand there in a speedo and be looked at like a f***ing hero"
+>
+-Kanye West
+
```
Comment (ifarah) [inline README.md:34]: **Coding style**  +
 **Coding strategy**   +
 **Presentation**  +
 **Achievement, creativity** +
 **Ease of access** +

It was great how you you effectively used five of the five style elements, photo, italics, headers, list, and link. Moreover, you included the caption, which showed mastery of the elements needed for the homework, and from this, I learned have to do this. It was nice getting to read your biography, and knowing a bit about you. Great job!

**Review by ifarah — COMMENTED** [review]
Peer review

**Comment by lmhjulian** [comment]
Coding style: Excellent
Coding strategy: Excellent
Achievement, creativity: Excellent
Ease of access: Excellent

This was a great piece of self introduction. It's clear from that you aware and are able to effectively use the various elements of Markdown. While not part of the requirements, you also managed to insert a quotation which showed the amount of effort you put into this assignment. Loved.The.Quote.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
 # hw01
-Repository for first CFSS homework assignment
+# Nora C. Nickels  
+
+### ![](https://0.academia-photos.com/30174041/13103076/14424279/s200_nora.nickels.png)
```
Comment (bensoltoff) [inline README.md:4]: Not sure why this was placed on a line with a header. Should just be

```
![](https://0.academia-photos.com/30174041/13103076/14424279/s200_nora.nickels.png) 
```

Headers only work with text.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,34 @@
...
+
+https://sites.google.com/site/noracnickels/
+
+https://scholar.google.com/citations?user=7XtEhe0AAAAJ&hl=en
+
+https://imb.uchicago.edu/directory/nora-nickels
+
+http://chicago.academia.edu/NoraNickels
+
+##
```
Comment (bensoltoff) [inline README.md:29]: Also could use `----` for a blank header line. Like below:

----

**Review by bensoltoff — COMMENTED** [review]
Evaluation HW-01 for Nora Nickels
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |     x         |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* Nice, simple biography. Makes frequent use of Markdown elements. Nice!
* Not sure why you had the issues pushing to GitHub. Typically this occurs when two separate clones of the repo are attempting to push to GitHub. You might have cloned the repo multiple times on your computer and accidentally switched between the two. It helps to organize all repos in a single folder on your computer. Make a folder just for this class, and stash all the repos there. It helps to prevent multiple copies from floating around on your computer

**Pull request closed without merging**